### ISSUE TITLE
Doesn't appear to work with Grafana 7.x

### ISSUE THREAD

**Issue by B3DTech** [kumaravel29/grafana-sankey-panel#31]
Grafana complains "Error loading plugin", but I'm not seeing specific errors in the log file.

I'm having the exact same issue as #23.  Same error messages, and installed it using:

grafana-cli --pluginUrl https://codeload.github.com/kumaravel29/grafana-sankey-panel/zip/master plugins install grafana-sankey-panel

It installed it to the right location and it says it was successful.  Restarted Grafana and still get the same errors.  Grafana log file doesn't show any errors though.

**Comment by panks21** [comment]
+1 on this... 
Not working on 7.4.3
Anyone with workaround??
![image](https://user-images.githubusercontent.com/18426368/109331746-3c246880-7883-11eb-9340-65cfcdab5d3a.png)

**Comment by kumaravel29** [comment]
@B3DTech and @panks21 , can you please check if the index.html is being updated with the google charts link https://github.com/kumaravel29/grafana-sankey-panel#info

**Comment by panks21** [comment]
I don't remember I updated the google charts link. Let me try again and report back

**Comment by shidaying** [comment]
@panks21 do you resolve this problem

**Comment by shidaying** [comment]
> @B3DTech and @panks21 , can you please check if the index.html is being updated with the google charts link https://github.com/kumaravel29/grafana-sankey-panel#info

@kumaravel29 what's the path of index.html? how check

**Comment by kumaravel29** [comment]
@shidaying , you should be able to find index.html at the following location - /usr/share/grafana/public/views/

**Comment by B3DTech** [comment]
It is working after updating the index.html file.
